website/docs/sigs.md

```diff
@@ -250,6 +250,46 @@ end
 Main.name_length(Main.greet('Alice')) # => error!
 ```
 
+## Class Methods
+
+There are at least three ways to define class (static) methods in ruby.
+The placement of the `extend T::Sig` varies accordingly.
+
+1. `def self.greet`
```

Comment:
Can we make it clear that this is vastly preferred to the other approaches?

Reply:
Sure, based on your other comments I would just say that (2) and (3) are not supported?